pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewSchema.java

```diff
@@ -54,37 +55,20 @@ public class ViewSchema {
    * @param fhirType the given FHIR type
    * @return the corresponding JDBCType; if `fhirType` is not mapped JDBCType.VARCHAR is returned.
    */
-  public static JDBCType fhirTypeToDb(String fhirType) {
+  public static JDBCType fhirTypeToDb(@Nullable String fhirType) {
     if (fhirType != null) {
-      switch (fhirType) {
-        case "boolean":
-          return JDBCType.BOOLEAN;
-        case "integer":
-        case "unsignedInt":
-          return JDBCType.INTEGER;
-        case "integer64":
-          return JDBCType.BIGINT;
-        case "decimal":
-          return JDBCType.DECIMAL;
-        case "date":
-          return JDBCType.DATE;
-        case "dateTime":
-        case "instant":
-          return JDBCType.TIMESTAMP;
-        case "time":
-          return JDBCType.TIME;
-        case "base64Binary":
-        case "canonical":
-        case "code":
-        case "id":
-        case "markdown":
-        case "oid":
-        case "string":
-        case "uri":
-        case "url":
-        case "uuid":
-          return JDBCType.VARCHAR;
-      }
+      return switch (fhirType) {
+        case "boolean" -> JDBCType.BOOLEAN;
+        case "integer", "unsignedInt" -> JDBCType.INTEGER;
+        case "integer64" -> JDBCType.BIGINT;
+        case "decimal" -> JDBCType.DECIMAL;
+        case "date" -> JDBCType.DATE;
+        case "dateTime", "instant" -> JDBCType.TIMESTAMP;
+        case "time" -> JDBCType.TIME;
+        default -> JDBCType
+            .VARCHAR; // also covers base64Binary, canonical, code, id, markdown, oid, string, uri,
```

Comment:
nit: consider moving the comment before `default` line.

Reply:
Done